SocialCareCaseViewerApi/V1/Boundary/Requests/UpdateFormSubmissionAnswersRequest.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using FluentValidation;
+using Microsoft.AspNetCore.Mvc;
+
+#nullable enable
+namespace SocialCareCaseViewerApi.V1.Boundary.Requests
+{
+    public class UpdateFormSubmissionAnswersRequest
+    {
+        [FromBody]
+        [JsonPropertyName("editedBy")]
+        public string EditedBy { get; set; } = null!;
+
+        [FromBody]
+        [JsonPropertyName("stepAnswers")]
+        public Dictionary<string, object>
```

Comment:
I'd love a way to represent this without using the object as the value.

Essentially this dictionary represents the questions and answers for a section of a form.
The question id acts as the key (typeof string) which is fine.
The answer can come in varying types from the [frontend](https://github.com/LBHackney-IT/lbh-social-care-frontend/blob/main/data/flexibleForms/forms.types.ts#L58).

The issue is the type could be a string, a list of strings, or even a list of dictionaries that map a key of type string to either a string or list of strings.

Using the value type as an object works as a "catch all" for these types but unintentionally allows basically any type. Maybe this is ok because we don't ever expect this to be called except for by the frontend (if people aren't being sneaky). However, it is worth noting that the answers that map to a question id could be of an invalid type.